docs/types.rst

```diff
@@ -54,7 +54,7 @@ Operators:
 * Bit operators: ``&``, ``|``, ``^`` (bitwise exclusive or), ``~`` (bitwise negation)
 * Arithmetic operators: ``+``, ``-``, unary ``-``, unary ``+``, ``*``, ``/``, ``%`` (remainder), ``**`` (exponentiation), ``<<`` (left shift), ``>>`` (right shift)
 
-Division always truncates (it just maps to the DIV opcode of the EVM), but it does not truncate if both
+Division always truncates (it just maps to the DIV opcode of the EVM. Maps will be explained later `[here](http://solidity.readthedocs.io/en/develop/types.html#mappings)`_), but it does not truncate if both
```

Comment:
Yeah sorry I've since figured out that it's mean to be of this format:

`[here](http://solidity.readthedocs.io/en/develop/types.html#mappings)`_